branchpro/tests/test_models.py

```diff
@@ -20,3 +21,35 @@ def test_simulate(self):
         forward_model = bp.ForwardModel()
         with self.assertRaises(NotImplementedError):
             forward_model.simulate(0, 1)
+
+
+class TestBrachProModelClass(unittest.TestCase):
+    """
+    Test the 'BranchProModel' class.
+    """
+    def test__init__(self):
+        branch_model = bp.BranchProModel(0, [1])
+        self.assertEqual(branch_model._serial_interval, np.array([1]))
+        self.assertEqual(branch_model._initial_r, 0)
+        self.assertEqual(branch_model._normalizing_const, 1)
+
+        with self.assertRaises(ValueError):
+            bp.BranchProModel(0, [0])
+
+        with self.assertRaises(TypeError):
+            bp.BranchProModel('0', [1])
+            bp.BranchProModel(0, 1)
```

Comment:
I thought you split this out?

Reply:
I did... I wonder why it is not appearing